Cargo.toml

```diff
@@ -371,3 +373,8 @@ opt-level = 1
 codegen-units = 4
 debug = "line-tables-only"
 lto = "fat"
+
+[patch.crates-io]
+# temporarily unblock bip39 conflict with iroh 0.35
+# We might need to publish it as a fork, if we can't figure out a better way.
+bip39 = { git = "https://github.com/rust-bitcoin/rust-bip39/", rev = "bc2376d110f963e8be64f521016b2fdaf9a8dcee" }
```

Comment:
This is release blocking, what was the conflict again?

Reply:
What ended up being the resolution that doesn't make this release blocking?

I'm a bit out-of-the-loop on the different threads associated with this PR, but it looks like https://github.com/rust-bitcoin/rust-bip39/pull/86 landed and was included in `2.2.0` (https://github.com/rust-bitcoin/rust-bip39/blob/v2.2.0/Cargo.toml#L53). If that's the case, are we good to change this to the upstream tag or do we still need to use this revision?